pydruid/utils/aggregators.py

```diff
@@ -22,16 +22,24 @@ def longsum(raw_metric):
     return {"type": "longSum", "fieldName": raw_metric}
 
 
+def longmin(raw_metric):
+    return {"type": "longMin", "fieldName": raw_metric}
+
+
+def longmax(raw_metric):
+    return {"type": "longMax", "fieldName": raw_metric}
+
+
 def doublesum(raw_metric):
     return {"type": "doubleSum", "fieldName": raw_metric}
 
 
-def min(raw_metric):
```

Comment:
Could you leave the old ones in for compatibility with older python clients? They should turn into `doubleMin` and `doubleMax` since the old min/max aggregators acted like the double versions.

Maybe also add a docstring to the old min/max versions saying they're deprecated.